Documentation/Installation/TutorialDdev.rst

```diff
@@ -136,13 +136,6 @@ Upon calling :bash:`ddev config` DDEV automatically created a database for
 you. DDEV also created a file called :file:`config/system/additional.php`
 in which it stored the database credentials for you.
 
-During the installation setup process TYPO3 created all the tables it needed.
```

Comment:
This line is still correct, isn't it?

Reply:
Yes, that was removed with the other by accident.

Reply:
ok, for me :)